macosx/Controller.mm

```diff
@@ -506,6 +506,17 @@ - (instancetype)init
         tr_variantDictAddInt(&settings, TR_KEY_peer_limit_global, [_fDefaults integerForKey:@"PeersTotal"]);
         tr_variantDictAddInt(&settings, TR_KEY_peer_limit_per_torrent, [_fDefaults integerForKey:@"PeersTorrent"]);
 
+        NSInteger bindPort = [_fDefaults integerForKey:@"BindPort"];
+        if (bindPort <= 0 || bindPort > 65535)
+        {
+            // First launch, we avoid a default port to be less likely blocked on such port and to have more chances of success when connecting to swarms.
+            // Ideally, we should be setting port 0, then reading the port number assigned by the system and save that value. But that would be best handled by libtransmission itself.
+            // For now, we randomize the port as a Dynamic/Private/Ephemeral Port from 49152–65535
+            // https://datatracker.ietf.org/doc/html/rfc6335#section-6
```

Comment:
Kind of a superficial comment, but ... are we not running clang-format checks on the macOS client?

I wouldn't expect it to let a 173-character-wide comment line pass unchallenged :dizzy:

Reply:
Seems that it's accepted by clang-format. Do we set a limit somewhere?
Comments wrap themselves on Xcode IDE, but yeah, they don't wrap on GitHub snippets.

Reply:
Line 1901 in the same file is already even longer.

Reply:
Sorry for not marking this thread as 'resolved' sooner. Leaving it as unresolved prevents merging & this is not a blocking concern